kernelci/build.py

```diff
@@ -1211,6 +1230,43 @@ def install(self, verbose=False):
         return super().install(verbose)
 
 
+class FetchFirmware(Step):
+
+    @property
+    def name(self):
+        return 'firmware'
+
+    def run(self, jopt=None, verbose=False, opts=None):
+        """Fetch linux-firmware repository
+
+        If kernel have CONFIG_EXTRA_FIRMWARE enabled we need to fetch
+        fresh snapshot of linux-firmware git repository, otherwise
+        kernel will not be able to build
+        """
+        # CONFIG_EXTRA_FIRMWARE_DIR need absolute path
+        full_path = os.path.abspath(self._output_path)
+        fwdir = os.path.join(full_path, 'linux-firmware')
+        key = self._kernel_config_getkey('CONFIG_EXTRA_FIRMWARE')
+        if key and len(key) < 3:
```

Comment:
Maybe it's worth adding a comment to a magic number `3`?

Reply:
Made it more readable (just if it is == '""')